src/global/intTypes.h

```diff
@@ -38,8 +38,8 @@ typedef std::vector<int32_t> Int32Vector;
 /// Typedef for Query ID in query metadata.
 typedef std::uint64_t QueryId;
 typedef std::int64_t JobId;
-typedef JobId UberJobId;           // These must be the same type.
-typedef std::uint32_t CzarIdType;  // TODO:UJ remove qmeta::CzarId and rename this CzarId
+typedef JobId UberJobId;  // These must be the same type.
```

Comment:
This feels wrong to me. The way you're thinking here is that both `JobId` and `UberJobId` are _implemented_ as `std::int64_t`, which is true. In reality, you have two separate _types_ representing separate series of identifiers. Unfortunately, `typedef` (neither `using`) introduces an alias for `std::int64_t`, which doesn't prevent mistakes in calling these functions using values from the wrong sequences:
```
void foo(JobId);
void bar(UberJobId);
```
Ideally, each sequence should be represented by its own data type. C++ allows this.  Unfortunately, this would require some design effort.